doc/sdk-cli-reference.md

```diff
@@ -265,6 +284,24 @@ Create deploy/crds/app_v1alpha1_appservice_crd.yaml
 Create deploy/crds/app_v1alpha1_appservice_cr.yaml
 ```
 
+## run
+
+### ansible
+
+Runs as an ansible operator process. This is intended to be used when running
+in a Pod inside a cluster.
```

Comment:
We should add a note that users should always use `operator-sdk up local` as opposed to this. And that this command is largely intended for the SDK's internal lifecycle purposes.

Both here and in the CHANGELOG.